docs/source/documentation.rst

```diff
@@ -183,6 +173,28 @@ Below is a short tutorial describing how to use the AxonDeepSeg plugin for Napar
 
    <iframe width="700" height="394" src="https://www.youtube.com/embed/zibDbpko6ko" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
+Updating AxonDeepSeg
+-------------------
+
+To update an already cloned AxonDeepSeg package, we reccommend that users delete their conda environment and re-do the installation 
+
+pull the latest version of the project from GitHub and re-install the application as documented above. 
+
+To uninstall a conda environment:::
+
+    conda env remove -n ads_venv
+
+In addition, we recommend that user (developers: see note below) delete the axondeepseg/ folder and redownload it as described in our installation instructions to ensure that your software will be the latest version.
+
+From the parent folder of the AxonDeepSeg directory:::
+
+    rm -rf axondeepseg
```

Comment:
For Windows users, the command will be `rmdir axondeepseg`

Reply:
Is that on the powershell? Would the command be the same on the Anaconda terminal?

Reply:
> Would the command be the same on the Anaconda terminal?

Yes